sources/packages/web/src/components/common/OfferingView.vue

```diff
@@ -70,38 +70,45 @@
                   )
                 }}
               </li>
+              <li v-if="!offeringViewData.studyBreaks">-</li>
```

Comment:
you can use `DEFAULT_EMPTY_VALUE`

Reply:
I have a question here. Are we going to keep it empty or use `-` ?

Reply:
@ann-aot  Also, I could not get a way to use `DEFAULT_EMPTY_VALUE` via `emptyStringFiller` or `conditionalEmptyStringFiller` here. Maybe I am missing something. Can you please propose how I can do it.

Reply:
usually, we follow the Figma. Since we don't have UX/UI, follow what we have on other pages. and if a the utility method is not there, then create new

Reply:
Reverted the change as per discussion in the call.